app/models/task.py

```diff
@@ -2,4 +2,21 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True, default=None)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goal.goal_id"), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
+
+
+    def to_dict(self):
+        task_as_dict = {}
+        task_as_dict["id"] = self.id
+        task_as_dict["title"] = self.title
+        task_as_dict["description"] = self.description
+        task_as_dict["is_complete"] = bool(self.completed_at)
+        if self.goal_id:
+            task_as_dict["goal_id"] = self.goal_id
+
+        return task_as_dict
```

Comment:
Remember to always put a newline at the end of the file -- this is good style practice in Python and a requirement in some other languages.